src/components/PopupWithForm.js

```diff
@@ -17,6 +19,10 @@ export default class PopupWithForm extends Popup {
     return formInputs;
   }
 
+  handleDeleteCardConfirmation(handleDeleteCardSubmit) {
+    this._handleFormSubmit = handleDeleteCardSubmit;
```

Comment:
`handleDeleteCardConfirmation` can't be in universal class `PopupWithForm`. Please, rename it in a neutral way: `setSubmitHandler`

`PopupWithForm` should not know about deleting or anything else. It's only about forms and modals